test/use_case/toggle_audio/ToggleAudioInteractorTest.java

```diff
@@ -0,0 +1,59 @@
+package use_case.toggle_audio;
+
+import data_access.InMemoryGameDataAccessObject;
+import entity.*;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ToggleAudioInteractorTest {
+    private Song song;
+    private Game game;
+    private ToggleAudioGameDataAccessInterface gameDataAccessObject;
+    @Before
+    public void init() {
+        song = new CommonSong("Closer", "The Chainsmokers", new OnlineMP3PlayableAudio("path/song.mp3"));
```

Comment:
While using `OnlineMP3PlayableAudio` will work for now since this class is not implemented properly, this will likely cause an error in testing once the actual implementation is done since `path/song.mp3` doesn't exist. I think we should create a `TestPlayableAudio` class or something similar that looks exactly like what you have in `OnlineMP3PlayableAudio` right now so that these tests will continue to work later.

Reply:
Really good point, I agree. Should I add a TestPlayableAudio class that implements the PlayableAudio interface the same way OnlineMP3Player currently does and put it under entities?

Reply:
Done